openverse_catalog/dags/providers/provider_api_scripts/smithsonian.py

```diff
@@ -1,531 +1,443 @@
 """
-Content Provider:  Smithsonian
+Content Provider:       Smithsonian
 
-ETL Process:       Use the API to identify all CC licensed images.
+ETL Process:            Use the API to identify all CC licensed images.
 
-Output:            TSV file containing the images and the respective
-                   meta-data.
+Output:                 TSV file containing the images and the respective meta-data.
 
-Notes:             None
+Notes:                  https://api.si.edu/openaccess/api/v1.0/search
 """
-import json
+
 import logging
-import os
-from datetime import datetime
 from typing import Tuple
 
 from airflow.exceptions import AirflowException
 from airflow.models import Variable
-from common.licenses import LicenseInfo
+from common import constants
+from common.licenses import get_license_info
 from common.loader import provider_details as prov
-from common.requester import DelayedRequester
-from common.storage.image import ImageStore
+from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
 from retry import retry
 
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Should the logging level be raised to info to merge?

Reply:
This is a leftover, thanks! I'll delete this line.